### DISCUSSION TITLE
Mastodon 3.5.17 last update and 500 Internal Server Errors

### DISCUSSION THREAD

**Discussion by Metztli** [mastodon/mastodon#29169] [General]
Although I can see the writing on the wall, i.e., upgrade to Mastodon 4.x.y, I just want to notify of the 500 Internal Server Errors which might correspond with PostgreSQL 14.10 errors. Sample log fragment follows:

Requests with error response codes
    500 Internal Server Error
       /api/v1/directory?limit=1: 6 Time(s)
       /api/v1/directory?order=active&local=true&limit=40&offset=0: 1 Time(s
       
--------------------- PostgreSQL (14.10) Begin ------------------------

 Errors:
 -------
 
 7 times:
 [2024-02-10 06:50:09 +/-12 hour(s)]  column reference "last_status_at" is ambiguous

**Comment by mjankowski** [comment]
Closing this because 3.5 is EOL ... were you able to upgrade?

**Reply by Metztli** [reply]
I apologize for not closing this stale issue 'a priori'.

Indeed, I upgraded to 4.xy. Thank you!
